### ISSUE TITLE
Should be able to override `aria-controls` for `<Tooltip>`

### ISSUE THREAD

**Issue by ivarni** [SpareBank1/designsystem#633]
#### Which package is this issue related to?

`ffe-form-react`

#### Describe your issue (screenshots welcome!)

The `Tooltip` component supports rendering just the toggle-button [without any children](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-form-react/src/Tooltip.js#L58). When it does, the `aria-controls` attribute set [here](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-form-react/src/Tooltip.js#L47) does not refer to anything existing in the DOM.

#### What is the expected behavior? 

We should never render an `aria-controls` attribute that reference a non-existing DOM element.

#### What is the actual behavior?

When the `Tooltip` is used in this way, the consumer should provide an `aria-controls` attribute. We might not be able to *force* them to do it, but they should at least have the option.

I propose that when no `children`  props is passed to `Tooltip` it does not set any `aria-controls` attribute on its button unless one is passed from the parent.
